bindings/nodejs/lib/client/client.ts

```diff
@@ -61,6 +61,9 @@ import { plainToInstance } from 'class-transformer';
 export class Client {
     private methodHandler: ClientMethodHandler;
 
+    /** 
+     * @param options TODO: what should we write here?
```

Comment:
Maybe the same as in client-method-handler.ts `client options or a client method handler.`